acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature

```diff
@@ -0,0 +1,86 @@
+@parallel=false
```

Comment:
If possible, new tests should adhere to the best practises and avoid this indicator

Reply:
Extracted existing reusable block as a separate functions. It's possible to farther improve them but it makes more sense to do this in scope of a separate ticket